adback-solution-to-adblock-lite/class-ad-back.php

```diff
@@ -36,6 +36,9 @@ public function getMyInfo()
             }
 
             $fullScripts = $this->askFullScripts();
+            $genericScript = $this->askGenericScripts();
+
+            $fullScripts['script_codes']['generic'] = $genericScript;
 
             $types = array(
                 'analytics',
```

Comment:
You should also add the iab banner script, it is the one which is going to display the ad

Reply:
Updated !